packages/typespec-ts/test/modularIntegration/generated/azure/core-traits/src/rest/outputModels.ts

```diff
@@ -1,15 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-/** Provides the 'If-*' headers to enable conditional (cached) responses */
-export interface ConditionalRequestHeadersOutput {}
-
-/** Provides the 'ETag' header to enable conditional (cached) requests */
-export interface EtagResponseEnvelopeOutput {}
-
-/** Provides the 'x-ms-client-request-id' header to enable request correlation in requests and responses. */
-export interface ClientRequestIdHeaderOutput {}
```

Comment:
these are just empty models and should not be generated and it is introduced by templateArgs logic.